_website/models.py

```diff
@@ -150,7 +150,8 @@ class Meta:
         ordering = ['-pk']
 
 class UsefulLink(models.Model):
-    title = models.CharField(max_length=300)
+    title_tr = models.CharField(max_length=300)
```

Comment:
don't you also need to change the templates

Reply:
Actually only need to change _models.Model_ to _MultilingualModel_.